src/olympia/migrations/989-replace-adminreview-with-addonreviewerflags.sql

```diff
@@ -0,0 +1,15 @@
+-- Make the old field nullable to prepare for its future removal.
+ALTER TABLE `addons` MODIFY COLUMN `adminreview` tinyint(1) NULL;
+-- Create the new table.
+CREATE TABLE `addons_addonreviewerflags` (
+    `created` datetime(6) NOT NULL,
+    `modified` datetime(6) NOT NULL,
+    `addon_id` integer NOT NULL PRIMARY KEY,
+    `needs_admin_code_review` bool NOT NULL,
+    `needs_admin_content_review` bool NOT NULL
+)
+;
+ALTER TABLE `addons_addonreviewerflags` ADD CONSTRAINT `addon_id_refs_id_7a280313` FOREIGN KEY (`addon_id`) REFERENCES `addons` (`id`);
```

Comment:
"addon_id_refs_id_7a280313" - a memorable constraint name 😉

Reply:
It's generated by django, so it sucks but it's a lot better than our custom ones that make us inconsistent with django syncdb/migrations :)